.env.dollar

```
@@ -1,5 +1,23 @@
-NEXT_PUBLIC_PROTOCOL_NAME="GLUON DOLLAR"
+NEXT_PUBLIC_FAVICON=/logo/gluon-green.png
 NEXT_PUBLIC_NEUTRON_TICKER=GSD
-NEXT_PUBLIC_PROTON_TICKER=GLYD
 NEXT_PUBLIC_PEG_ASSET=USD
 NEXT_PUBLIC_PEG_UNIT="1 USD"
+NEXT_PUBLIC_PROTON_TICKER=GLYD
+NEXT_PUBLIC_PROTOCOL_NAME="GLUON DOLLAR"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_PROTOCOL_NAME key should go before the NEXT_PUBLIC_PROTON_TICKER key

(UnorderedKey)